pkg/virt-api/webhooks/validating-webhook/admitters/admitters_suite_test.go

```diff
@@ -3,9 +3,15 @@ package admitters
 import (
 	"testing"
 
+	v1 "kubevirt.io/api/core/v1"
+
 	"kubevirt.io/client-go/testutils"
 )
 
 func TestValidatingWebhook(t *testing.T) {
 	testutils.KubeVirtTestSuiteSetup(t)
 }
+
+func vmiSpecUsedStub(_ *v1.VirtualMachineInstanceSpec) bool {
```

Comment:
It will be cleaner if you inline it as a lambda function:

```go
VmiSpecUsed: func(_ *v1.VirtualMachineInstanceSpec) bool { return true },
```

Reply:
Done

Reply:
I do not see it changed.